spec/integration/type/file_spec.rb

```diff
@@ -1260,6 +1260,27 @@ def build_path(dir)
     end
 
     let(:source) { tmpfile_with_contents("source_default_values", "yay") }
+
```

Comment:
Didn't care for my spec tests at https://github.com/MikaelSmith/puppet/commit/045eed3a2d0c584ba90340ee57b651c91b1a14a5#diff-1a4c70e8a84e98497af8fcdbc8403474R1283?